WORKSPACE.bazel

```diff
@@ -28,6 +28,28 @@ nixpkgs_flake_package(
     package = "mkosi",
 )
 
+nixpkgs_flake_package(
+    name = "org_openssl",
+    build_file_content = """\
+load("@rules_cc//cc:defs.bzl", "cc_library")
+filegroup(
+    name = "include",
+    srcs = glob(["include/**/*.h"]),
+    visibility = ["//visibility:public"],
+)
+cc_library(
+    name = "org_openssl",
+    srcs = glob(["lib/**/*.a"]),
+    hdrs = [":include"],
+    strip_include_prefix = "include",
+    visibility = ["//visibility:public"],
+)
+""",
+    nix_flake_file = "//:flake.nix",
+    nix_flake_lock_file = "//:flake.lock",
+    package = "openssl",
```

Comment:
openssl has a .dev output. Do we need the full output?

Reply:
This is using the `openssl` from flake.nix that combines multiple outputs. The `.dev` output only contains headers. We need the combined `.dev` (headers) and `.out` (compiled library) in one derivation for Bazel to be happy.